manifests/params.pp

```diff
@@ -32,6 +34,7 @@
 
     'Archlinux': {
       $dhcp_dir    = '/etc'
```

Comment:
I just realized we manage `/etc` on Arch. That doesn't sound right. Perhaps a `Boolean $manage_dhcp_dir` parameter makes sense. Same for the BSDs.

Reply:
That's what I intended by the condition, but I guess this PR doesn't make it worse.
Let's open a PR for adding that parameter?